packages/core/admin/server/src/controllers/webhooks.ts

```diff
@@ -20,7 +21,23 @@ const urlRegex =
 const webhookValidator = yup
   .object({
     name: yup.string().required(),
-    url: yup.string().matches(urlRegex, 'url must be a valid URL').required(),
+    url: yup
+      .string()
+      .matches(urlRegex, 'url must be a valid URL')
+      .required()
+      .test(
+        'is-public-url',
+        "Url is not supported because it isn't reachable over the public internet",
+        async (url) => {
```

Comment:
Aren't we supposed to prevent this only in production env ?

Reply:
oops, yes! I added the if case

Reply:
🤔  I think we might want !== production in case we are doing tests for ex

Reply:
done!